src/site/apt/index.apt.vm

```diff
@@ -72,6 +72,8 @@ ${project.name}
 
   *{{{./list-mojo.html}dependency:list}} alias for resolve that lists the dependencies for this project.
 
+  *{{{./list-classes-mojo.html}dependency:list-classes}} displays the fully package qualified names of all classes found in a specified dependency.
```

Comment:
Shouldn't that read `package-qualified`?

Reply:
damned if I know. Added hyphen.